docker-compose.yml

```diff
@@ -1,85 +1,163 @@
+      - DATABASE_PORT=5432
+      - REDIS_HOST=redis
+      - REDIS_PORT=6379
+      # comma separated list, no brackets, e.g. localhost,dev.myhost.com
+      - DJANGO_SETTINGS_MODULE=osmaxx_conversion_service.config.settings.production
+      - NUM_WORKERS=5
+    depends_on:
+      - conversionserviceredis
+      - mediatordatabase
+    networks:
+      - osmaxx-internal
+  worker:
+    image: geometalab/osmaxx-worker:${DEPLOY_VERSION}
+    extends:
+      file: docker-compose-common.yml
+      service: conversionbase
+    command: [honcho, -f, ./osmaxx_conversion_service/Procfile.worker, start]
+    volumes:
+      - osm_data:/var/data/osm-planet
+      - worker-data:/data/media/job_result_files
+    links:
+      - conversionserviceredis:redis
+    env_file:
+      - ./compose-env/worker.env
+    environment:
+      - REDIS_HOST=redis
+      - REDIS_PORT=6379
+      - DJANGO_SETTINGS_MODULE=osmaxx_conversion_service.config.settings.worker
+    depends_on:
+      - conversionserviceredis
+    networks:
+      - osm-world
+      - osmaxx-internal
+  conversionserviceredis:
+    image: redis
+    networks:
+      - osmaxx-internal
+  mediatordatabase:
+    image: geometalab/postgis:9.4
+    volumes:
+      - mediator-database-data:/database/data
+    environment:
+      - PGDATA=/database/data
+    networks:
+      - osmaxx-internal
+  ##### CONVERSION SERVICE END ########
+  ##### WORLD-DB START ########
+  world-database:
+    # to limit scaling to exactly 1, we need to specify the container name
+    container_name: osmworlddb_postgis
+    image: geometalab/osm_world_postgis
+    volumes:
+      - database-postgis-data:/var/lib/postgresql
+    command: postgres -B 12GB -F -N 512 -S 20GB --max_wal_size=5GB
+    environment:
+      - POSTGRES_DB=gis
+      - POSTGRES_USER=gis
+      - POSTGRES_PASSWORD=gis
+    networks:
+      - osm-world-internal
+      - osm-world
+    logging:
+      driver: "json-file"
+      options:
+        max-size: 50k
+  osm_importer:
+    # to limit scaling to exactly 1, we need to specify the container name
+    container_name: osmworlddb_osm_importer
+    image: geometalab/osm2pgsql:latest
+    entrypoint: /root/wait-for-it.sh database:5432 --
+    command: /root/run.sh
+    links:
+      - world-database:database
+    depends_on:
+      - world-database
+    volumes:
+      - osm_data:/var/data/osm-planet
+      - osm_cache:/var/cache/osm-cache
+    environment:
+      - num_processes=16
+    networks:
+      - osm-world-internal
+    logging:
+      driver: "json-file"
+      options:
+        max-size: 50k
+  ##### WORLD-DB END ########
+volumes:
+  # osmaxx
+  frontend-database-data: {}
+  frontend-data: {}
+  frontend-results: {}
+  mediator-database-data: {}
+  worker-data: {}
+  osmplanet-data: {}
+  # osm-world-db
+  osm_data: {}
+  osm_cache: {}
+  database-postgis-data: {}
+networks:
+  osmaxx-internal:
+    driver: bridge
+  osm-world:
+    driver: bridge
+  osm-world-internal:
+    driver: bridge
```

Comment:
This network layout separates some containers from each other, but still let's containers access each other that would not have to. One could consider splitting the `osmaxx-internal` network into 5 separate networks:
- network `osmaxx-frontend-internal` between containers
  - `frontend`
  - `frontenddatabase`
- network `osmaxx-mediator-internal` between containers
  - `mediator`
  - `mediatordatabase`
- network `osmaxx-conversion-rest-api` (or `osmaxx-core`? name T.B.D.) between containers
  - `frontend`
  - `mediator`
- network `osmaxx-conversionqueue-push` (or a better name T.B.D.) between containers
  - `mediator`
  - `conversionserviceredis`
- network `osmaxx-conversionqueue-pull` (or a better name T.B.D.) between containers
  - `worker`
  - `conversionserviceredis`

I've discussed that with @hixi and we came to the conclusions:
- Accidental unwarranted access between containers is sufficiently deterred by the need to link containers first. (One can deliberately work around that, but not unintentionally.)
- Neither network layout adds much to the application security, as the sensitive data is held in the front end and its data base.
- System security isn't inherently increased by the layout with more networks: If someone manages to break out of docker, they can probably do that from the front end already.

So for now, we'll stick with the network layout introduced in this PR.